cli/commands/health/health.go

```diff
@@ -0,0 +1,67 @@
+package cluster
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/sensu/sensu-go/cli/elements/table"
+	"github.com/sensu/sensu-go/types"
+	"github.com/spf13/cobra"
+)
+
+func HealthCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "health",
+		Short:        "get sensu health status",
+		SilenceUsage: false,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			result, err := cli.Client.Health()
+			if err != nil {
+				return err
+			}
+			return helpers.Print(cmd, cli.Config.Format(), printHealthToTable, nil, result)
+		},
+	}
+	return cmd
+}
+
+func printHealthToTable(result interface{}, w io.Writer) {
+	table := table.New([]*table.Column{
```

Comment:
Same thing here, maybe check that `clusterHealth, ok := data.(*types.ClusterHealth)` is ok since we depend on that in each column?